### PR TITLE
Enable mirroring corenode for all non pki nodes

### PR DESCRIPTION
**Pull request by ianopolous**

### CODE REVIEW

File: src/peergos/server/tests/MultiNodeNetworkTests.java [hunk 1]
```diff
@@ -92,7 +94,7 @@ public static void init() throws Exception {
                     .with("social-sql-file", ":memory:");
             Main.PEERGOS.main(normalNode);
 
-
+            Thread.sleep(10_000);
```
Comment (kevodwyer) [inline src/peergos/server/tests/MultiNodeNetworkTests.java]: why the sleep() call

Comment (ianopolous) [reply src/peergos/server/tests/MultiNodeNetworkTests.java]: Only way I could get the ipfs swarm connect calls not to error all the time.

Comment (ianopolous) [reply src/peergos/server/tests/MultiNodeNetworkTests.java]: (the ipfs daemon, despite being able to answer id calls over http, can't correctly answer this call yet)

File: src/peergos/shared/mutable/HttpMutablePointers.java [hunk 2]
```diff
@@ -18,14 +18,29 @@
 
     private static final boolean LOGGING = true;
     private final HttpPoster direct, p2p;
+    private final String directUrlPrefix;
```
[the remark this answers is not in the document]
Comment (ianopolous) [reply src/peergos/shared/mutable/HttpMutablePointers.java]: Yep! That's done in a later pr.

File: src/peergos/server/corenode/MirrorCoreNode.java [hunk 3]
```diff
@@ -33,6 +32,10 @@ public MirrorCoreNode(CoreNode writeTarget,
         this.ipfs = ipfs;
         this.mutable = mutable;
         this.pkiOwnerIdentity = pkiOwnerIdentity;
+    }
+
+    public void start() {
+        running = true;
```
[the remark this answers is not in the document]
Comment (ianopolous) [reply src/peergos/server/corenode/MirrorCoreNode.java]: This method is never called outside tests. If there are problems like you mention we will fix them.

**Pull request merged**